packages/agents/router/src/router.ts

```diff
@@ -97,10 +98,23 @@ export const makeRouter = async () => {
     }
     await bindServer();
     await bindMetrics();
-    await bindSubgraph(context.config.subgraphPollInterval!);
+    await bindSubgraph();
     await bindCache();
 
-    logger.info("Router ready!");
+    logger.info("Bindings initialized.");
+
+    console.log(
```

Comment:
Why is this not logger.info?

Reply:
I can make the message logger.info, but the connext branding thing has to be console.log or it literally will just show up incoherently in the logs haha